src/System/Taffybar/Widget/Windows.hs

```diff
@@ -62,6 +70,8 @@ defaultWindowsConfig =
   WindowsConfig
   { getMenuLabel = truncatedGetMenuLabel 35
   , getActiveLabel = truncatedGetActiveLabel 35
+  , showActiveIcon = True
+  , getActiveWindowIconPixbuf = defaultGetWindowIconPixbuf
```

Comment:
Instead of a Bool AND a getter, why not just a `Maybe WindowIconPixbufGetter`

Reply:
Also a better name would be getWindowIconPixbuf. This pixbuf getter could also be used in e.g. the menu.

Reply:
Converted to `Maybe WindowIconPixbufGetter`.
We can't easily use the record name `getWindowIconPixbuf` because it conflicts with the one from `WorkspacesConfig`.